### ISSUE TITLE
no endpoint to get ticket under an user

### ISSUE THREAD

**Issue by simarsingh24** [fossasia/open-event-server#5037]
**Is your feature request related to a problem? Please describe.**
One should be able to get list of tickets under an user

**Comment by srv-twry** [comment]
@simarsingh24 
Do you just need to show the tickets under an user or more information also, for e.g. the attendee profile/order etc. 

The FE uses the following code to fetch everything.
```JavaScript
return this.get('authManager.currentUser').query('attendees', {
      include : 'ticket,event,order',
      filter  : filterOptions
    });
```

So basically, get all the attendees under that user and ticket from that attendee.
If the requirement is only to show the tickets, no other information then I can add this endpoint if you want.

**Comment by simarsingh24** [comment]
@srv-twry yes please, there should be a dedicated endpoint to get tickets under an user.
(only showing tickets no other info)

**Comment by srv-twry** [comment]
@simarsingh24 I did some research on this. I think you don't the tickets under a user. There no such thing called tickets under a user. 
what you need is the orders under a user i.e. the orders submitted by the user. You can get the tickets under that user from those orders. Please check this PR: https://github.com/fossasia/open-event-server/pull/5048

**Comment by simarsingh24** [comment]
@srv-twry Yes orders will do thanks
